tests/Sitemap/SitemapPageTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Website\Tests\Sitemap;
+
+use DateTimeImmutable;
+use Doctrine\Website\Sitemap\SitemapPage;
+use PHPUnit\Framework\TestCase;
+
+class SitemapPageTest extends TestCase
+{
+    /** @var string */
+    private $url;
+
+    /** @var DateTimeImmutable */
+    private $date;
+
+    /** @var SitemapPage */
+    private $sitemapPage;
+
+    public function testGetUrl() : void
+    {
+        self::assertSame('/test.html', $this->sitemapPage->getUrl());
+    }
+
+    public function testGetDate() : void
+    {
+        self::assertSame($this->date, $this->sitemapPage->getDate());
+    }
+
+    protected function setUp() : void
+    {
+        $this->url  = '/test.html';
+        $this->date = $this->createMock(DateTimeImmutable::class);
```

Comment:
Should we use a test double for all `DateTimeImmutable` classes, where it's value doesn't matter in the tests?